sensor_msgs/Image image_stereo
bool reloadmap
---
float32 mapnum

